src/main/java/com/task/lecture9/controller/VinylController.java

```diff
@@ -38,36 +37,24 @@ public List<Vinyl> getVinyls() {
     }
 
     @GetMapping("/vinyls/{id}")
-    public Object getVinylById(@Validated @PathVariable(required = false) Integer id) throws Exception {
+    public Object getVinylById(@PathVariable(required = false) Integer id) throws Exception {
         return vinylService.findById(id);
     }
 
     @PostMapping("/vinyls")
-    public ResponseEntity<Map> create(@RequestBody InsertForm insertForm, HttpServletRequest request) {
+    public ResponseEntity<Map> create(
```

Comment:
下記のようにMapのkey/valueの型を明示したいです。
```suggestion
    public ResponseEntity<Map<String, String>> create(
```